adapters/taboola/taboola.go

```diff
@@ -221,12 +229,30 @@ func createTaboolaRequests(request *openrtb2.BidRequest) (taboolaRequests []*ope
 		modifiedRequest.BAdv = taboolaExt.BAdv
 	}
 
+	if taboolaExt.PageType != "" {
+		modifiedRequest.Ext = makeRequestExt(taboolaExt.PageType, errs)
+	}
+
```

Comment:
@jennylt @ahmadlob 
made the change so that pageType will be in requestExt instead of siteExt